tests/end2end_tests/tests/service-container-sshnpd.yaml

```diff
@@ -5,7 +5,7 @@
     networks:
       - sshnpd
     healthcheck:
-      test: ["CMD", "grep", "-Eq", "monitor started for @", "/atsign/err.txt"]
```

Comment:
What's the reason for not suppressing output?

Reply:
I added back `-q`